nnpackage/spec/10_packaging_and_manifest.md

```diff
@@ -84,9 +91,24 @@ Here is an example of `MANIFEST`.
 ```
 {
     "major-version" : "1",
-    "minor-version" : "0",
+    "minor-version" : "1",
     "patch-version" : "0",
+    "configs"     : [ "model.cfg" ],
     "models"      : [ "mymodel.model", "yourmodel.model" ],
     "model-types" : [ "tflite", "circle" ]
 }
 ```
+
+## 5. Configuration file
+
+Configuration file is a human readable plain text file having one `key=value` in each line.
+- `#` is used as comment and will be ignored afterwards.
+- all leading and trailing white spaces will be ignored in both `key` and `value`.
+
+For example
+```
+BACKENDS=cpu
+ EXCUTOR=Linear # some comment
```

Comment:
```suggestion
EXCUTOR=Linear # some comment
```

Reply:
This was intentionally added with a space to check trim works OK

Reply:
I think adding a comment would help